examples/lib/scheduled-function.ts

```diff
@@ -1,16 +1,13 @@
 import { BillingMode } from '@aws-cdk/aws-dynamodb';
 import cdk = require('@aws-cdk/core');
-import { Duration } from '@aws-cdk/core';
-import { Schedule } from '@aws-cdk/aws-events';
-
-import { DynamoDB, integer, Lambda, string } from 'punchcard';
+import punchcard = require('punchcard');
+import { string, integer } from 'punchcard';
 
 const app = new cdk.App();
 export default app;
 
 const stack = new cdk.Stack(app, 'scheduled-function-example');
-
-const table = new DynamoDB.HashTable(stack, 'my-table', {
+const table = new punchcard.HashTable(stack, 'my-table', {
```

Comment:
I think you have an old version, I moved everything into service-namespaces in https://github.com/sam-goodwin/punchcard/pull/26